cmd/fs-v1.go

```diff
@@ -1306,6 +1306,11 @@ func (fs *FSObjects) IsNotificationSupported() bool {
 	return true
 }
 
+// IsListenBucketSupported returns whether listen bucket notification is applicable for this layer.
+func (fs *FSObjects) IsListenBucketSupported() bool {
+	return true
```

Comment:
This still enables ListenBucket you have to override this in NAS @poornas by adding another method.

Reply:
done.